session/session.go

```diff
@@ -222,7 +222,7 @@ func (s *Session) SetBaseTickDelay(baseTickDelay time.Duration) {
 	}
 }
 
-const provSearchDelay = time.Second * 10
```

Comment:
This is pretty low. What was the reasoning behind this change?

Reply:
To match time time for regular GetBlocks in Bitswap w/o sessions... especially if we're going to merge these two.

Reply:
But transparently in this case, for testing. And it should be configurable.

Reply:
> To match time time for regular GetBlocks in Bitswap w/o sessions... especially if we're going to merge these two.

The timer for GetBlocks is a bit different. That's an *initial* delay before finding a provider for a single block. In the past, we didn't even delay this. Subsequent FindProviders calls are controlled by the `rebroadcastWorker` and sent out every 10 seconds (for random blocks).

Reply:
yea in this case, after a block is received, it switches over to using the tick delay, which is 500ms + 3 * average latency per block. So, somewhat similar.

Reply:
provSearchDelay is only used at the beginning of a session for the first call to FindProvidersAsync, if no blocks come back from any peers.